### PR TITLE
Port openlibm erf

### PR DESCRIPTION
**Pull request by tgymnich**

### CODE REVIEW

File: src/initialization.jl [hunk 1]
```diff
@@ -43,6 +43,18 @@ function __init__()
     if isdefined(Base, :active_repl_backend)
         push!(Base.active_repl_backend.ast_transforms, synchronize_metal_tasks)
     end
+
+    # register device overrides
+    if !precompiling
+        eval(Expr(:block, overrides...))
+        empty!(overrides)
```
[the remark this answers is not in the document]
Comment (tgymnich) [reply src/initialization.jl]: fixed

[the remark this answers is not in the document]
Comment (tgymnich) [reply src/initialization.jl]: using an extension now

[the remark this answers is not in the document]
Comment (christiangnrd) [reply src/initialization.jl]: Seems like these lines (and lines 54-55) returned after my pull request was merged

[the remark this answers is not in the document]
Comment (maleadt) [reply src/initialization.jl]: This PR isn't merged yed?

[the remark this answers is not in the document]
Comment (maleadt) [reply src/initialization.jl]: It's still there, but I'll clean it up afterwards.

**Pull request merged**